### PR TITLE
Fix some test files 

### PR DESCRIPTION
**Pull request by dalinaum**
* Add omitted copyright headers.
* createAndTestFilename public -> private
* remove an unused variable, get_data
* [Use assertEquals(expected, actual, delta) to compare floating-point numbers. assertEquals(expected, actual) for floating-pointer numbers is deprecated in JUnit.](http://junit.org/apidocs/org/junit/Assert.html#assertEquals(double, double))

### CODE REVIEW

**Comment by dalinaum** [comment]
It's a small patch. Please review this @realm/java

**Comment by dalinaum** [comment]
retest this please

File: realm/realm-library/src/androidTest/java/io/realm/RealmTest.java [hunk 1]
```diff
@@ -71,6 +71,7 @@
 public class RealmTest extends AndroidTestCase {
 
     protected final static int TEST_DATA_SIZE = 10;
+    protected final static double DELTA = 1e-15;
```
Comment (cmelchior) [inline realm/realm-library/src/androidTest/java/io/realm/RealmTest.java]: why not 0? Since I assume we want to test `get(set(x)) = x`?

Comment (dalinaum) [inline realm/realm-library/src/androidTest/java/io/realm/RealmTest.java]: I thought 1e-15 is enough big, but I agree that 0 is appropriate.

**Comment by beeender** [comment]
:+1:

**Comment by cmelchior** [comment]
:+1:

**Comment by kneth** [comment]
:+1:

**Pull request merged**